src/main/java/frc/robot/commands/Turret/VisionSpinTurret.java

```diff
@@ -17,14 +19,23 @@ public class VisionSpinTurret extends CommandBase {
   Turret turret;
   Shooter shooter;
   Vision vision;
+  NavX navX;
 
   double target;
+  double oldTargetPosition = 0;
+  double oldNavXPosition = 0;
+  double newTargetPosition = 0;
+  double changeInNavx = 0;
+  double oppositePosition = 0;
+  boolean is360Spin = false;
 
   /** Creates a new VisionSpinTurret. */
-  public VisionSpinTurret(Turret sub_turret, Shooter sub_shooter, Vision sub_vision) {
+  public VisionSpinTurret(Turret sub_turret, Shooter sub_shooter, Vision sub_vision, NavX sub_navX) {
```

Comment:
This command is almost identical to `VisionAimTurret` so this command should be deleted